### ISSUE TITLE
M304 not saved restart printer

### ISSUE THREAD

**Issue by michtill** [mcheah/Marlin4MPMD#53]
Marlin4MPMD v133

SENDING:M304
echo: p:80.00 i:12.71 d:300.00
SENDING:M304 P66.69 I12.63 D88.02
echo: p:66.69 i:12.63 d:88.02
SENDING:M500
SENDING:M304
echo: p:66.69 i:12.63 d:88.02
--- NOW RESETING PRINTER MANALY :) ---
Disconnected.
Printer is now online.
SENDING:M304
echo: p:80.00 i:12.71 d:300.00

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Software: Pronterface, Repetier Host

**Comment by mcheah** [comment]
Yup, I can confirm that saving the bed PID values is not implemented currently.  You should not need to change these values unless you've drastically modified your printer however.
